README.md

```diff
@@ -28,7 +28,7 @@ Please don't use Github issues for feature requests, instead create a topic in t
 
 ### License
 
-Copyright 2014 Hummingbird Media, Inc.
+Copyright 2015 Hummingbird Media, Inc.
```

Comment:
The answer is [oh](https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html) [god](http://www.photoattorney.com/qa-what-year-should-i-use-for-my-copyright-notice/) [fuck](http://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated) [copyright](http://www.copyrightlaws.com/copyright-basics/copyright-notice-year/) [law](http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/).

Also, it really doesn't matter because under the Berne Convention it's all copyrighted anyways: the notice is really just a formality